recipes/mfem/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -x
+
+make config CXX=$CXX PREFIX=$PREFIX MFEM_SHARED=YES MFEM_STATIC=NO MFEM_USE_MPI=YES CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS" LIBS="$LIBS" HYPRE_OPT="-I$PREFIX/include"
+
+cat config/config.mk
+
+make lib -j${CPU_COUNT}
```

Comment:
Is there a `make check` or `make test`?

Reply:
I added a small serial test (based on `make check`).

Reply:
Thanks. What happened with the parallel test? I think you'd need https://github.com/conda-forge/mpi4py-feedstock/blob/master/recipe/meta.yaml#L44-L48 for openmpi.

Reply:
Thanks, that fixed it! Reenabled the parallel test.